tensorflow/python/compiler/tensorrt/utils.py

```diff
@@ -45,3 +48,19 @@ def disable_non_trt_optimizers_in_rewriter_config(rewriter_config):
   rewriter_config.remapping = off
   rewriter_config.scoped_allocator_optimization = off
   rewriter_config.shape_optimization = off
+
+
+def _IsTensorRTVersionGreaterEqual(_trt_ver, major, minor=0, patch=0):
+  assert isinstance(_trt_ver, tuple)
+  assert len(_trt_ver) == 3
+
+  _trt_ver = [str(x) for x in _trt_ver]
+  _trt_ver = LooseVersion(".".join(_trt_ver))
+  _target_ver = LooseVersion("{}.{}.{}".format(major, minor, patch))
```

Comment:
parameter names and local variable names can't start with _
s/_trt_ver/trt_ver/
s/_target_ver/target_ver/

Reply:
Fixed